src/components/sortControl/sortControl.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { describe, expect, test } from 'vitest';
+import {
+  screen,
+  render,
+  fireEvent,
+  getByTestId,
+  waitFor,
+} from '@testing-library/react';
+import SortControl from './SortControl';
+import user from '@testing-library/user-event';
+import { vi } from 'vitest';
+
+let sortOptions: movieSortOption[];
+describe('Genre Select tests', () => {
```

Comment:
Please check also:
- the options are hidden by default
- the options are hidden after selection